content/en/docs/tasks/run-application/run-single-instance-stateful-application.md

```diff
@@ -163,8 +168,8 @@ specific to stateful apps:
 Delete the deployed objects by name:
 
 ```
-kubectl delete deployment,svc mysql
-kubectl delete pvc mysql-pv-claim
+kubectl delete -f https://k8s.io/docs/tasks/run-application/mysql-deployment.yaml
+kubectl delete -f https://k8s.io/docs/tasks/run-application/mysql-pv.yaml
```

Comment:
For deletion, maybe the original commands are better. Fetching the yaml files only for deletion is an overkill.